validator/validator-main.protoascii

```diff
@@ -5321,6 +5320,39 @@ tags: {
   }
 }
 
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "IMG"
+  spec_name: 'I-AMPHTML-INTRINSIC-SIZER'
+  mandatory_parent: "I-AMPHTML-SIZER"
+  attrs: {
+    name: "class"
+    value_regex: "i-amphtml-intrinsic-sizer"
```

Comment:
can this just be `value:`?

Note that in validation rules, value_regex always means full match and blacklisted_value_regex is partial match. This is a bit confusing, but it is in place to avoid mistakes in the wrong direction (looser validation rules than expected). 


If you intended partial match, use ".*i-amphtml-intrinsic-sizer.*" and also ignore my next comment.

Reply:
First I had it as value, but that caused the test to fail. I couldn't work out why. I assumed that was specific to how `class` is handled:

```
    |      <i-amphtml-sizer class="i-amphtml-sizer"><img alt="" aria-hidden="true" class="i-amphtml-intrinsic-sizer" role="presentation" src="data:image/svg+xml;charset=utf-8,<svg height=&quot;100&quot; width=&quot;300&quot; xmlns=&quot;http://www.w3.org/2000/svg&quot; version=&quot;1.1&quot;/>"></i-amphtml-sizer>
    >>                                              ^~~~~~~~~
    transformed_feature_tests/server_side_rendering.html:40:45 The mandatory attribute 'class' is missing in tag 'IMG-I-AMPHTML-INTRINSIC-SIZER'. [DISALLOWED_HTML]
```

Reply:
Thats odd. Is the example you give the result for that test? Did you verify the spelling?